specutils/analysis/flux.py

```diff
@@ -193,7 +193,17 @@ def _compute_equivalent_width(spectrum, continuum=1, regions=None,
     dx = np.abs(np.diff(spectrum.spectral_axis.bin_edges))
     ew = np.sum((1 - line_flux / cont_flux) * dx)
 
-    return ew.to(spectrum.spectral_axis.unit)
+    # NOTE: uncertainty gets lost during unit translation, so we have to convert first
```

Comment:
Is there an issue for this?  I didn't realize this, definitely needs to be fixed if we can! (not part of this PR though)

Reply:
I don't think there's an issue filed.  @pllim made the good point that dropping custom attributes added to a quantity might actually be desired behavior as keeping them might suggest they were properly handled during the conversion (for example, if you set a float to `quantity.uncertainty` and then do math on that object, would you expect the float to remain fixed or react somehow to the math?).  But I do think this limitation is a reason that we should expose the uncertainties in a more "officially supported" way.